.github/ISSUE_TEMPLATE/usage_question.yaml

```diff
@@ -0,0 +1,66 @@
+name: Usage Question
+description: Ask a question
+labels: ["Type: usage"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        While we enable issues as a mechanism for new contributors and passers-by who are unfamiliar with Apache Software Foundation projects to ask questions and interact with the project, we encourage users to ask such questions on public mailing lists:
+        
+        * Development discussions: dev@arrow.apache.org (first subscribe by sending an e-mail to dev-subscribe@arrow.apache.org).
+
+        * User discussions: user@arrow.apache.org (first subscribe by sending an e-mail to user-subscribe@arrow.apache.org).
+        
+        Do not be surprised by responses to issues raised here directing you to those mailing lists, or to report a bug or feature request here.
+
+        Thank you!
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the usage question you have. Please include as many useful details as possible.
+    validations:
+      required: true
+  - type: dropdown
+    id: component
+    attributes:
+      label: Component
+      description: Which component does your question relate to?
+      options:
+        - Archery
+        - Benchmarking
+        - C
+        - C#
+        - C++
+        - C++ - Gandiva
+        - C++ - Plasma
+        - Compute IR
+        - Continuous Integration
+        - Dart
+        - Developer Tools
+        - Documentation
+        - FlightRPC
+        - Format
+        - GLib
+        - Go
+        - GPU
+        - Integration
+        - Java
+        - JavaScript
+        - Julia
+        - MATLAB
+        - Packaging
+        - Parquet
+        - Python
+        - R
+        - Ruby
+        - Rust
+        - Rust - Ballista
+        - Rust - DataFusion
```

Comment:
Should we even list rust/julia or add a link to `arrow-rs|jl` to  'Contact Links' in `config.yml`?

Reply:
Fair question, though I will note this is a very direct port of existing component behavior found in ASF Jira.